.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # the repo. Unless a later match takes precedence,
 # following users will be requested for
 # review when someone opens a pull request.
-*  @fm-sick @vervoortb @haliCR1 @BaRu-ifm @julian-st @lorandmolnar @mrentsch65
+*  @fm-sick @vervoortb @haliCR1 @BaRu-ifm @julian-st @lorandmolnar @mrentsch65 @roeseko
```

Comment:
Maybe this user does not have write access according to the error message

Reply:
oh, he wasn't a project member yet, I added him.